source/globalCommands.py

```diff
@@ -272,7 +272,7 @@ def script_reportCurrentSelection(self,gesture):
 	)
 	def script_dateTime(self,gesture):
 		if scriptHandler.getLastScriptRepeatCount()==0:
-			text=winKernel.GetTimeFormatEx(winKernel.LOCALE_NAME_USER_DEFAULT, winKernel.TIME_NOSECONDS, None, None)
+			text=winKernel.GetTimeFormatEx(winKernel.LOCALE_NAME_USER_DEFAULT, None, None, None)

```

Comment:
Fix lint

```suggestion
			text = winKernel.GetTimeFormatEx(winKernel.LOCALE_NAME_USER_DEFAULT, None, None, None)
```

Reply:
I can of course fix this lint, one thing that confuses me is that this function alone contains multiple lints that need to be fixed. Do I just need to fix this line that was changed?
There are many places in this globalCommands.py that need to be fixed by lint, Do we need to deal with it uniformly in the future?

Reply:
Pull requests require that the diff of the changes you make is linted.
So in this case, yes just this line that was changed.
When editing the majority of a function, it may be worth while to lint and add typing to the whole function.
However with small changes making large linting changes is noisy and distracting.